metadata-ingestion/src/datahub/sdk/datajob.py

```diff
@@ -285,3 +305,29 @@ def _set_browse_path_from_flow(self, flow: DataFlow) -> None:
         browse_path.append(models.BrowsePathEntryClass(id=flow.name, urn=str(flow.urn)))
         # Set the browse path aspect
         self._set_aspect(models.BrowsePathsV2Class(path=browse_path))
+
+    @property
+    def inlets(self) -> List[str]:
```

Comment:
let's make this + outlets a list of urn types?

Reply:
updated